twake/backend/node/jest.config.js

```diff
@@ -0,0 +1,10 @@
+// jest.config.js
+// Sync object
+/** @type {import('@jest/types').Config.InitialOptions} */
+const config = {
+  verbose: true,
+  rootDir: ".",
+  preset: "ts-jest"
+};
+
+module.exports = config;
```

Comment:
Why is this file needed ? (just to know)

Reply:
For requires by JetBrains Idea for running tests in debug mode (very convenient for TDD)

Reply:
I'll add it to .gitignore better